docs/examples/csfle-automatic_encryption-local_schema.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use MongoDB\BSON\Binary;
+use MongoDB\Client;
+use MongoDB\Driver\ClientEncryption;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/';
+
+// Generate a secure local key to use for this script
+$localKey = new Binary(random_bytes(96));
+
+/* Create a client with no encryption options. Additionally, create a
+ * ClientEncryption object to manage data keys. */
+$client = new Client($uri);
+
+$clientEncryption = $client->createClientEncryption([
+    'keyVaultNamespace' => 'encryption.__keyVault',
+    'kmsProviders' => [
+        'local' => ['key' => $localKey],
+    ],
+]);
+
+/* Drop the key vault collection and create an encryption key. Alternatively,
+ * this key ID could be read from a configuration file. */
+$client->selectCollection('encryption', '__keyVault')->drop();
+$keyId = $clientEncryption->createDataKey('local');
+
+/* Create a client with automatic encryption enabled. Specify a schemaMap option
+ * to enforce a local JSON schema. */
+$encryptedClient = new Client($uri, [], [
+    'autoEncryption' => [
+        'keyVaultNamespace' => 'encryption.__keyVault',
+        'kmsProviders' => ['local' => ['key' => $localKey]],
+        'schemaMap' => [
+            'test.coll' => [
+                'bsonType' => 'object',
+                'properties' => [
+                    'encryptedField' => [
+                        'encrypt' => [
+                            'keyId' => [$keyId],
+                            'bsonType' => 'string',
+                            'algorithm' => ClientEncryption::AEAD_AES_256_CBC_HMAC_SHA_512_DETERMINISTIC,
+                        ],
+                    ],
+                ],
+            ],
+        ],
+    ],
+]);
+
+// Drop and create the collection.
+$encryptedClient->selectDatabase('test')->dropCollection('coll');
```

Comment:
I guess you drop the collection in case the script runs several time on the same db, that should be explained.

Reply:
I can revise these comments to explain that I'm doing so to make the scripts idempotent (not sure if that's the correct term to use here, but certainly able to be run multiple times w/o error).

Reply:
You may group all the cleanup operations so that the rest of the example is simpler and can be copy-pasted.

Or do the cleanup at the end of the example, so in reenforce the understanding on what have been created and clean the server when running tests.

Reply:
The setup operations depend on client objects that are created in the context of the script, so I don't think there's a good way to segregate that. And for some scripts, collection creation uses encryption options.

I purposely omitted all cleanup operations from the scripts, as I can handle that ExamplesTest.php. The drop commands are really setup operations to ensure we're starting with a fresh slate.

I'm amenable to revisiting this later if we end up adding more inline code examples (much of the PHPLIB docs omit examples for API reference), but I'll note that our docs example in the MongoDB manual (DocumentationExamplesTest.php) don't demonstrate setup/cleanup at all -- which makes them a bit less useful for users IMO.